src/components/Home.js

```diff
@@ -1,15 +1,22 @@
-import React, { useState } from 'react'
+import React from 'react'
 import { v4 as uuid4 } from 'uuid'
 
 import AddQuiz from './AddQuiz'
 uuid4()
-export default function Home() {
-  const [quizzes, setQuizzes] = useState([])
 
-  const addQuiz = (quiz) => {
-    // spread operator makes a copy of currrent quiz and we're adding id,
+class Home extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      quizzes: [],
+    }
+  }
+
+  addQuiz = (quiz) => {
+    // spread operator makes a copy of current quiz and we're adding id
     const newQuiz = [
```

Comment:
`newQuiz` holds an array of quizzes, so a more appropriate name would be something like `updatedQuizzes`